trin-core/src/types/receipts.rs

```diff
@@ -307,11 +392,45 @@ mod tests {
             ],
         };
         assert_eq!(
-            hex::encode(transactions.root().unwrap()),
+            hex::encode(receipts.root().unwrap()),
             EXPECTED_RECEIPTS_ROOT.to_owned()
         );
     }
 
+    #[test]
+    fn ssz_encoding_decoding_receipts() {
+        let receipts = Receipts {
+            receipt_list: vec![
+                Receipt::decode(&hex::decode(RECEIPT_0).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_1).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_2).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_3).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_4).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_5).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_6).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_7).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_8).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_9).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_10).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_11).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_12).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_13).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_14).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_15).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_16).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_17).unwrap()).unwrap(),
+                Receipt::decode(&hex::decode(RECEIPT_18).unwrap()).unwrap(),
+            ],
+        };
+        let encoded = receipts.as_ssz_bytes();
+
+        let expected = std::fs::read_to_string("./src/types/assets/receipts_14764013.hex").unwrap();
```

Comment:
Oh, and after re-encoding the file, it should probably have a suffix other than `.hex`. Maybe... `.data` or `.bin`?